frontend/src/components/pages/GlobalCodeReviewDashboard/FilterByNames.jsx

```diff
@@ -2,20 +2,43 @@ import React from "react";
 import FormGroup from "@material-ui/core/FormGroup";
 import FormControlLabel from "@material-ui/core/FormControlLabel";
```

Comment:
Please make a FilterByNames directory. It should have an index.jsx file and a presentation.jsx file.